### ISSUE TITLE
[IxCategoryFilter] Make category filter without categories controllable

### ISSUE THREAD

**Issue by robert-wettstaedt** [siemens/ix#508]
### Suggestion / feature request

As per the [docs](https://ix.siemens.io/docs/controls/category-filter#without-categories), the category filter has a mode "without categories" where it is possible to only suggest string options. For this mode the component is not [controllable](https://react.dev/learn/sharing-state-between-components#controlled-and-uncontrolled-components) meaning there is no way to control the selected suggestions. The `filterState` prop that controls the component in the "with categories" mode gets ignored in the "without categories" mode.

**Comment by nuke-ellington** [comment]
@robert-wettstaedt thank you for your request. What exactly are you trying to do? Settings filter tokens without categories should work. Maybe you can even post some example code?

**Comment by robert-wettstaedt** [comment]
@nuke-ellington [here is an example](https://codesandbox.io/s/make-category-filter-without-categories-controllable-sz95rw). Notice how I cannot control the selected value of the `IxCategoryFilter`. When the `filterChanged` event fires, I set a state to the selected value. After a small timeout I want to clear the value and thereby control the component's current selection. As far as I understand this would work in the "with categories" mode but not in the "without categories" mode.

This is basically what the `IxSelect` component allows, but I chose to use `IxCategoryFilter` because it implements a `inputChanged` event when the user searches the list of items. `IxSelect` currently does not allow that for now, so I also created a FR for that right here: #572
